src/containers/flows/MakeABid/MakeABid.tsx

```diff
@@ -407,12 +422,17 @@ const MakeABid: React.FC<MakeABidProps> = ({ domain, onBid }) => {
 					{details()}
 				</div>
 				<div className={styles.InputWrapper}>
-					{domain.owner.id.toLowerCase() === account?.toLowerCase() && (
+					{isDomainOwner && (
 						<p className={styles.Error} style={{ paddingTop: '16px' }}>
 							You can not bid on your own domain
 						</p>
 					)}
-					{domain.owner.id.toLowerCase() !== account?.toLowerCase() && (
+					{!isDomainOwner && !isBiddable && (
+						<p className={styles.Error} style={{ paddingTop: '16px' }}>
+							You can not bid on this domain because it is disabled bid
```

Comment:
I think we should change this slightly if you agree...

You can not bid on this domain because it is disabled bid -> Bidding has been disabled on this domain

Reply:
Updated the text